src/backend/common/helpers/tests/match_helper_test.py

```diff
@@ -271,6 +273,118 @@ def test_upcoming_matches_all_played(test_data_importer) -> None:
     assert upcoming_matches == []
 
 
+def _parse_match_schedule_csv(event, filename):
+    matches = []
+    with open(filename) as f:
+        reader = csv.DictReader((line.lower() for line in f), skipinitialspace=True)
+        for row in reader:
+            alliances = {}
+            for alliance_color in ("red", "blue"):
+                alliances[alliance_color] = {
+                    "teams": [
+                        "frc" + row[f"{alliance_color} {i + 1}"] for i in range(3)
+                    ],
+                    "score": row.get(f"{alliance_color} score", -1),
+                }
+
+            match = Match(
+                event=event.key,
+                id=Match.render_key_name(
+                    event.key_name,
+                    row["comp_level"],
+                    row["set_number"],
+                    row["match_number"],
+                ),
+                comp_level=row["comp_level"],
+                set_number=int(row["set_number"]),
+                match_number=int(row["match_number"]),
+                alliances_json=json.dumps(alliances),
+                time_string=row["time"],
+            )
+
+            matches.append(match)
+
+    return matches
+
+
+def test_add_match_times(test_data_importer):
```

Comment:
These first two tests came from `./old_py2/tests/test_add_match_times.py`. They relied on the old `UsfirstMatchesParser`, which hasn't been ported, to read the test data, so I decided moving it over to CSV would be simpler. Other tests are using `test_data_importer.parse_match_list()` -> `MatchConverter.dictToModel_v3()` to read test matches, but that does not appear to handle the `time_string` field, and I don't need the majority of fields here anyway. But open to other ideas that don't involve writing a custom parser for the tests.

Also, is the `UsfirstMatchesParser` likely to return? I am considering deleting the old `test_add_match_times`; wondering if I should also delete the corresponding test data (html files in `old_py2/test_data/usfirst_html/`)

Reply:
The HTML pages are long gone from FIRST's website, so I doubt we'll need them ever again.